src/main/java/org/kuali/kfs/module/ar/batch/service/impl/VerifyBillingFrequencyServiceImpl.java

```diff
@@ -0,0 +1,167 @@
+//KualiCo Patch Release 2020-02-13
+//FINP-4769 FK-117 changes applied.
```

Comment:
I like these comment, but if you could add what kualiCo patch will have these changes, so we know when the overlay file can be removed

This thought applies to ther

Reply:
Added patch FINP is in to seven files in question.